VoodooI2C/VoodooI2C/VoodooI2CController/VoodooI2CController.hpp

```diff
@@ -103,6 +103,14 @@ class EXPORT VoodooI2CController : public IOService {
      */
 
     IOReturn mapMemory();
+    
```

Comment:
From CircleCI:
```
VoodooI2C/VoodooI2CController/VoodooI2CController.hpp:106:  Line ends in whitespace.  Consider deleting these extra spaces.  [whitespace/end_of_line]
```

You should delete extra whitespace

Reply:
I think the lint tool wants this line to be removed.